lib/protocol/endpoint.js

```diff
@@ -29,11 +29,15 @@ module.exports = function(dependencies) {
 
     this._acquiredStreamSlots = 0;
     this._maximumStreamSlots = 0;
+    this._lastSuccessPingedTime = null;
+    this._pingedThreshold = (this.options.heartBeat || 60000) * 2;
+    this._heartBeatInterval = (this.options.heartBeat || 60000);
```

Comment:
oh, thanks for reminder, totally missed this from @jpwilliams. Updated and Committed